impl/src/lib.rs

```diff
@@ -12,12 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-pub mod bazel;
 pub mod context;
-pub mod license;
+pub mod error;
 pub mod metadata;
-pub mod planning;
-pub mod rendering;
+pub mod planning {
+  mod checks;
+  mod crate_catalog;
+  mod license;
+  mod planning;
+  mod subplanners;
+  pub use planning::*;
+}
+pub mod rendering {
+  pub mod bazel;
+  mod rendering;
```

Comment:
I think we should avoid this weird rendering::rendering thing if possible. Use `mod.rs` instead?

Reply:
it is, I didn't know if that would be more offensive to people (though I feel it's the correct thing to do)